actors/builtin/miner/miner_actor.go

```diff
@@ -1628,21 +1628,22 @@ func (a Actor) ApplyRewards(rt Runtime, params *builtin.ApplyRewardParams) *abi.
 type ReportConsensusFaultParams = miner0.ReportConsensusFaultParams
 
 func (a Actor) ReportConsensusFault(rt Runtime, params *ReportConsensusFaultParams) *abi.EmptyValue {
-	// Note: only the first reporter of any fault is rewarded.
```

Comment:
can we update this to explain why subsequent calls fail? That was tricky to figure out.